config/spinnaker.yml

```diff
@@ -294,3 +294,18 @@ providers:
       name: my-docker-registry-account
       address: https://index.docker.io/
       repository: library/nginx
+
+  openstack:
+    # For more information on configuring Openstack, see
+    # TODO Added doc link when available
+
+    enabled: false
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl:
+      username:
```

Comment:
If you are using environment variables in default-spinanker-local, you can put them here instead (e.g. projectName: ${SPINNAKER_OPENSTACK_PROJECT:}
That would be safer because really spinnaker-local should just contain the minimal overrides. I'd like to get rid of default-spinnaker-local or at least prune it down to just the minimal things that *have* to be overwritten. It was intended to be a minimal summary but minimal isnt so small anymore so might as well defer to spinnaker.yml. You can leave what you have up there for consistency with everything else, but prepare for it not being there.